package.json

```diff
@@ -9,10 +9,14 @@
     "serve": "vitepress serve pages",
     "deploy:config:copy": "mv ./pages/.vitepress/dist ./dist",
     "deploy:player:copy": "bash copy-player.sh",
-    "deploy": "rm -rf dist && yarn build && yarn deploy:config:copy && yarn deploy:player:copy && gh-pages -d dist"
+    "deploy:gh-pages": "gh-pages -d dist",
+    "deploy": "run-s  build deploy:config:copy deploy:player:copy deploy:gh-pages",
+    "pinstall": "yarn install --modules-folder ./player",
```

Comment:
Please make these match the naming conventions we have.

Example: install:player

Reply:
oops